docs/products/kafka/reference/kafka-metrics-prometheus.md

```diff
@@ -2,518 +2,546 @@
+
+## How to retrieve metrics
+
+To get the complete list of available metrics for your service, query the
+Prometheus endpoint. Follow these steps to get started:
+
+1. Ensure you have the following information:
+
+   - Aiven project certificate: `ca.pem`. To download the CA certificate, see
+     [Download CA certificates](/docs/platform/concepts/tls-ssl-certificates#download-ca-certificates).
+   - Prometheus credentials: `<PROMETHEUS_USER>:<PROMETHEUS_PASSWORD>`
+   - Aiven for Apache Kafka hostname: `<KAFKA_HOSTNAME>`
+   - Prometheus port: `<PROMETHEUS_PORT>`
+
+1. Use the curl command to query the Prometheus endpoint:
+
+   ```bash
+      curl --cacert ca.pem \
+        --user '<PROMETHEUS_USER>:<PROMETHEUS_PASSWORD>' \
+         'https://<KAFKA_HOSTNAME>:<PROMETHEUS_PORT>/metrics'
+   ```
+
+For detailed instructions on setting up Prometheus integration with
+Aiven, see [Use Prometheus with Aiven](/docs/platform/howto/integrations/prometheus-metrics)
 
 ## CPU utilization
 
--   `cpu_usage_guest`: CPU time spent running a virtual CPU for guest
-    operating systems.
--   `cpu_usage_guest_nice`: The amount of time the CPU runs a virtual
-    CPU for a guest operating system, which is low-priority and can be
-    interrupted by other processes. This metric is measured in
-    hundredths of a second.
--   `cpu_usage_idle`: Time the CPU spends doing nothing.
--   `cpu_usage_iowait`: Time waiting for I/O to complete.
--   `cpu_usage_irq`: Time servicing interrupts.
--   `cpu_usage_nice`: Time running user-niced processes.
--   `cpu_usage_softirq`: Time servicing softirqs.
--   `cpu_usage_steal`: Time spent in other operating systems when
-    running in a virtualized environment.
--   `cpu_usage_system`: Time spent running system processes.
--   `cpu_usage_user`: Time spent running user processes.
--   `system_load1`: System load average for the last minute.
--   `system_load15`: System load average for the last 15 minutes.
--   `system_load5`: System load average for the last 5 minutes.
--   `system_n_cpus`: Number of CPU cores available.
--   `system_n_users`: Number of users logged in.
--   `system_uptime`: Time for which the system has been up and running.
+CPU utilization metrics offer insights into CPU usage. These metrics
+include time spent on different processes, system load, and overall uptime.
+
+| Metric                  | Description                                                                                           |
+|-------------------------|-------------------------------------------------------------------------------------------------------|
+| `cpu_usage_guest`       | CPU time spent running a virtual CPU for guest operating systems.                                     |
+| `cpu_usage_guest_nice`  | CPU time spent running low-priority virtual CPUs for guest operating systems. These processes can be interrupted by higher-priority tasks. Metric is measured in hundredths of a second. |
```

Comment:
Except for this specific row, I'd remove all the final full stops from the `description` column coz they are not sentences. This comment refers to all the tables in this article wherever applicable.